morpheus/stages/general/monitor_stage.py

```diff
@@ -250,13 +252,15 @@ def _progress_sink(self, x):
 
         # Skip incase we have empty objects
         if (self._determine_count_fn is None):
-            return
+            return x
```

Comment:
Could we switch to something other than 'x' going forward, maybe `data`.

Reply:
Moving forward we can enforce using `data`. Thats better.